tests/e2e_logging.py

```diff
@@ -1630,23 +1630,34 @@ def test_committed_index(network, args):
 
     _, log_id = network.txs.get_log_id(txid)
 
-    r = network.txs.request(log_id, priv=True, url_suffix="committed")
+    remaining_retries = 10
+    while remaining_retries > 0:
```

Comment:
All our test retries are time based, rather than count based, I would prefer if we stayed consistent.